gateways/mercadopago/accesstoken/accesstoken_test.go

```diff
@@ -0,0 +1,29 @@
+package accesstoken
+
+import (
+	"fmt"
+	"testing"
+	"paygo/gateways/mercadopago"
+)
```

Comment:
is common in go we separate the oficial packages from external packages import with an empty line:

```
import (
       "fmt"
       "testing"

      "paygo/gateways/mercadopago"
)
```